### DISCUSSION TITLE
No access to images as guest?

### DISCUSSION THREAD

**Discussion by Kf637** [requarks/wiki#6957] [Help / Questions] [answered]
I have this issue where users that's not logged in can't see images, the page is public but seems like the images isn't. Can anyone help?
Logged in: 
![image](https://github.com/requarks/wiki/assets/43268197/6ff4b64e-3a15-45dc-b811-3467bc83e60b)

Not logged in: 
![image](https://github.com/requarks/wiki/assets/43268197/2a498491-6cca-4f57-97e1-f7a4e699b3b9)
![image](https://github.com/requarks/wiki/assets/43268197/679cd5fd-e744-40a0-9132-2cbda4dfad92)

**Comment by fnafox** [comment, marked as answer]
Guest user needs to have permissions to read assets you are trying to access.
My suggestion is to make a /pub/ directory with full guest read access and using it for any content that needs to be accessible without login.

**Reply by Kf637** [reply]
Thanks.
